common-custom-user-data-gradle-plugin/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 - Add custom value and search link for GitHub Actions run ID.
 - Fix generation of search links for custom values (#187).
```

Comment:
I think we are inconsistent in our linking to issues from the release notes items. Maybe we should jus stop doing that.  (for Gradle and Maven)

Reply:
Reference to issue removed.